my/heartbox.scad

```diff
@@ -0,0 +1,37 @@
+/// Create a Heart box with 2 circle and a square + minkowski function
+
+$fa = 0.01;
+$fs = 0.5;
+heart_box(20,15,0.5);
```

Comment:
Rather than putting this in a module, it's more elegant to directly write the code that creates it. The file itself serves as a container to semantically separate the code.